components/tabs/src/sync/bridge.rs

```diff
@@ -80,29 +79,29 @@ impl BridgedEngine for BridgedEngineImpl {
         handle_error! {
             let new_id = SyncGuid::random().to_string();
             let new_coll_ids = CollSyncIds {
-                global: SyncGuid::empty(),
-                coll: new_id.clone().into(),
+                global: new_id.clone().into(),
```

Comment:
I think this change is wrong - it's `coll` that we want to set, not `global`. This implies a test is probably missing - we should probably have a test that after calling `reset_sync_id`, `sync_id` returns the value we "reset" it to?